packages/lesswrong/components/ea-forum/EAHomeRightHandSide.tsx

```diff
@@ -236,6 +239,54 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontWeight: 600,
     fontSize: 13,
   },
+  givingSeason: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+    gap: "12px",
+    width: "100%",
+    maxWidth: 280,
+    background: theme.palette.grey[0],
+    borderRadius: theme.borderRadius.default,
+    padding: 16,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontSize: 16,
+    fontWeight: 700,
+    color: theme.palette.givingPortal[1000],
+    marginBottom: 32,
+  },
+  givingSeasonAmount: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "flex-start",
+    gap: "8px",
+    width: "100%",
+    color: theme.palette.grey[1000],
+    fontSize: 14,
+  },
+  givingSeasonProgress: {
+    background: theme.palette.givingPortal[900],
+    borderRadius: theme.borderRadius.small,
+    overflow: "hidden",
+    width: "100%",
+    height: 8,
+    "& *": {
+      height: "100%",
+      background: theme.palette.givingPortal[1000],
```

Comment:
We might want to use some of the `homepageHeader` colors here, so that things like the progress bar look red in dark mode still. The white is kind of confusing.

<img width="284" alt="Screen Shot 2023-11-21 at 11 02 51 AM" src="https://github.com/ForumMagnum/ForumMagnum/assets/9057804/82282ef5-e1fa-4487-a4f0-8cc62e9af48d">